UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/SalesCatalogue/ProductCounts.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace UKHO.ExchangeSetService.Common.Models.SalesCatalogue
+{
+    public class ProductCounts
+    {
+        public int? RequestedProductCount { get; set; }
+        public int? ReturnedProductCount { get; set; }
+        public int? RequestedProductsAlreadyUpToDateCount { get; set; }
+
+        public List<RequestedProductsNotReturned> RequestedProductsNotReturned { get; set; }
+    }
+
+    public class RequestedProductsNotReturned
+    {
+        public string ProductName { get; set; }
+        public string Reason { get; set; }
+    }
+}
```

Comment:
If somebody was looking for the RequestedProductsNotReturned class it would be easier to find if it was on its own file. Are these two classes grouped together for a reason?

Reply:
Did respective changes. Moved all sub class to individual class

Reply:
Thanks